### PR TITLE
Add Service Name header to be added to each page within the registrat…

### PR DESCRIPTION
**Pull request by lvnhmd**
…ion journey

### CODE REVIEW

File: packages/forms-web-app/src/views/layouts/register-main.njk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends "layouts/main.njk" %}
```
Comment (pk2453) [inline packages/forms-web-app/src/views/layouts/register-main.njk:1]: Is this file needed, or any issue with main.njk?

**Review by pk2453 — COMMENTED** [review]
I think modify serviceName in config.js and then add this line into app.js env.addGlobal('serviceName', config.serviceName); should be enough

**Review by cpcundill — APPROVED** [review]

File: packages/forms-web-app/src/views/layouts/register-main.njk (hunk 1 above)
Comment (lvnhmd) [reply packages/forms-web-app/src/views/layouts/register-main.njk:1]: @pk2453 [As per the ticket](https://pins-ds.atlassian.net/browse/ASB-162), we only need the service name in the register journey pages, main.njk is included in other pages as well
